frontend/src/components/Projects/MyProject.js

```diff
@@ -50,6 +51,15 @@ class MyProjects extends React.Component {
                       }
                     }}
                   >
+                    {/*<Link
+                    key={p.projectId}
+                    to={{
+                      pathname: "/ProjPanel",
```

Comment:
Would this be uncommented code be use eventually? otherwise i will suggest to remove it.

Reply:
Might be used in future, but almost I will remove it in future branches.